src/index.html

```diff
@@ -19,9 +19,367 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/reset.css"
```

Comment:
The HTML file is linking to an SCSS file (`styles/index.scss`). Browsers cannot interpret SCSS, so make sure to compile your SCSS files to CSS before linking them in your HTML.